data_structures/src/chain.rs

```diff
@@ -2888,6 +2880,12 @@ impl ReputationEngine {
             .total_active_reputation(|| self.calculate_active_rep())
     }
 
+    pub fn get_reputation(&self, pkh: &PublicKeyHash) -> u32 {
+        self.threshold_cache
+            .borrow_mut()
+            .get_trapezoid_reputation(pkh, || self.calculate_active_rep())
```

Comment:
Is this possible?
```suggestion
            .get_trapezoid_reputation(pkh, self.calculate_active_rep)
```

Reply:
![image](https://user-images.githubusercontent.com/44392885/90500259-f1df6380-e14a-11ea-9fed-08eee7a60b12.png)

Reply:
Oh yeah, sorry for the nonsense. Methods must be used as `ReputationEngine::calculate_active_rep` , but there's no way to capture `self` without the closure, so let's simply forget about this :sweat_smile: